CMakeLists.txt

```diff
@@ -694,4 +694,8 @@ endif()
 message(STATUS "")
 if(PY2_WARNING)
   message(WARNING "Python2 support will be dropped soon! We strongly advise migrating to Python3 as of now.")
+  find_python_module(pathlib)
+  if(NOT PATHLIB_FOUND)
+    message(SEND_ERROR "'pathlib' Python2 package is required. Please install it.")
+  endif()
```

Comment:
what if we just give warning and disable generation of docs string? (large number of people won't look at this?)

Reply:
discussed offline: will disable documentation if python2 pathlib not there, keeping this as warning